Basic_rBoot/app/application.cpp

```diff
@@ -95,7 +95,7 @@ void ShowInfo() {
 void serialCallBack(Stream& stream, char arrivedChar, unsigned short availableCharsCount) {
 	
 
-	if (arrivedChar == '\n') {
+	if (arrivedChar == '\r') {
```

Comment:
why have you made this change? not all systems will send a \r so this is the wrong thing to check for